lib/index.js

```diff
@@ -30,8 +30,16 @@ var Iterable = module.exports = integration('Iterable')
  */
 
 Iterable.prototype.track = function(payload, fn){
+  var endpoint;
+  if (payload["isUpdateCart"]) {
```

Comment:
Other question still stands though: However, I only want addedProduct to map to something special for us if they provide the full ```products``` (I think your spec doesn't require it). Otherwise I just want it to go to a regular track. Is there an easy way to accomplish that?